node-fs-readfile/cat.js

```diff
@@ -0,0 +1,16 @@
+import { readFile } from 'node:fs/promises';
+
+const [, , ...argList] = process.argv;
+
+async function readAny(fileName) {
+  try {
+    const file = await readFile(fileName, 'utf-8');
+    await console.log(file);
```

Comment:
`console.log` does not return a Promise. What happens when you `await` something that isn't a Promise? (https://deepscan.io/docs/rules/await-non-promise)
(You've made this error several times in your other code.)